src/SFA.DAS.EAS.Portal.UnitTests/SFA.DAS.EAS.Portal.UnitTests.csproj

```diff
@@ -9,9 +9,11 @@
 
```

Comment:
It'd be nice if we organised the unit tests in top level folders representing which project the unit tests pertain to, i.e. Portal, Worker, Client etc.

Reply:
Good idea.

Reply:
Unit test project structure now matches the source of the code under test.